cmd/juju/controller/config.go

```diff
@@ -243,7 +248,15 @@ func (c *configCommand) setConfig(client controllerAPI, ctx *cmd.Context) error
 	values := make(map[string]interface{})
 	for k := range attrs {
 		if controller.AllowedUpdateConfigAttributes.Contains(k) {
-			values[k] = parsed[k]
+			if field, ok := fields[k]; ok {
```

Comment:
Should we log/print a warning here like we do for model config if there's a controller attribute that is mistyped. I would even argue that we go a step further and make it an error. Is there any circumstance where a controller config attribute name is not defined in the schema?

Reply:
There is two schemas, only time this could happen is if it isn't in both.

Reply:
also you can't edit a config key if it isn't in `controller.AllowedUpdateConfigAttributes`